akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala

```diff
@@ -88,6 +89,13 @@ abstract class ActorContextSpec extends ScalaTestWithActorTestKit with AnyWordSp
     "canonicalize behaviors" in {
       val unhandledProbe = createTestProbe[UnhandledMessage]()
       system.eventStream ! EventStream.Subscribe(unhandledProbe.ref)
+      // make sure subscription completed before moving on
+      unhandledProbe.awaitAssert {
+        val uh = UnhandledMessage("dummy", akka.actor.ActorRef.noSender, unhandledProbe.ref.toClassic)
+        system.eventStream ! EventStream.Publish(uh)
+        unhandledProbe.expectMessageType[UnhandledMessage] shouldBe theSameInstanceAs(uh)
+      }
```

Comment:
Searching for "eventStream ! EventStream.Subscribe" shows that we have such testing in many places. Would it be worth adding some testkit utility for this?
```
val unhandledProbe: TestProbe[UnhandledMessage] = testKit.createUnhandledMessageProbe()
```

Reply:
Mmm, not a bad idea. I thought a bit about extending the event stream protocol with an subscribe+ack while working on this one. But thinking about it again it pretty much only matters in a test context where you have a single thread subscribing and then publishing/asserting.